lib/std/child_process.zig

```diff
@@ -30,10 +30,20 @@ pub const ChildProcess = struct {
 
     allocator: mem.Allocator,
 
+    /// The write end of the child process's standard input stream.
```

Comment:
We write to and read from, so `write end to child process's standard input stream` and `read end from child process's standard output stream` reads more correct.
See also below.

Reply:
I disagree, since here "write end" and "read end" are objects, similar to "the end _of_ a book", "the left end _of_ the stick". (We could also call them "writing end" and "reading end".)
They are just qualifying attributes specifying which "end", not verbs of the sentence. (Subject and verb would be "This (field) is", which are omitted at the beginning.)

Reply:
Thanks. I like "writing end" over "write end".
Another level of ambiguity may be addressed by calling it an end of the pipe rather than the stream, since the pipe is known to have two ends.
I'll take another go at it.